lib/odbc_adapter/database_statements.rb

```diff
@@ -128,7 +128,7 @@ def nullability(col_name, is_nullable, nullable)
     end
 
     def prepared_binds(binds)
-      prepare_binds_for_database(binds).map { |bind| _type_cast(bind) }
+      binds.map(&:value_for_database).map { |bind| _type_cast(bind) }
```

Comment:
i'm not following this change here -- what's wrong with the `prepare_binds_for_database_method`?

Reply:
I don't remember exactly what prompted it, but it's an internal method that's been removed from Rails:

https://github.com/rails/rails/commit/fb30df9c10372b1667dabac2fffe2cff25302626